wolfssl/wolfcrypt/port/cavium/cavium_octeon_sync.h

```diff
@@ -0,0 +1,79 @@
+/* cavium_octeon.h
+ *
+ * Copyright (C) 2006-2019 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL. (formerly known as CyaSSL)
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
```

Comment:
Please change to version 2 license.

Reply:
Fixed, and updated the filename at the top of the file.